servicetalk-examples/docs/modules/ROOT/pages/http/index.adoc

```diff
@@ -298,6 +298,11 @@ Async `Hello World` example that demonstrates how retry can be requested for a s
 * link:{source-root}/servicetalk-examples/http/retry/src/main/java/io/servicetalk/examples/http/retry/RetryUrlClient.java[RetryUrlClient.java] -
 Async `Hello World` example that demonstrates how retry can be requested for a multi-address client.
 
+[#TrafficResiliency]
+= Traffic Resiliency
+
+Some examples that use the traffic resiliency features.
```

Comment:
We typically clarify what those examples demonstrate and link to them. In this case, there are no clarifications on docs page nor javadoc in each example class.